lnd.go

```diff
@@ -1664,6 +1671,7 @@ func initNeutrinoBackend(cfg *Config, chainDir string) (*neutrino.ChainService,
 			return ips, nil
 		},
 		AssertFilterHeader: headerStateAssertion,
+		BlockCache:         blockCache.Cache,
```

Comment:
So with this, we'll now have double cache reads/writes when running under neutrino (one pair done by lnd, the other by neutrino itself). It shouldn't have an effect, but just noting it so that we're aware.